### DISCUSSION TITLE
output specific field in csv / json output

### DISCUSSION THREAD

**Discussion by lijiejie31** [projectdiscovery/httpx#1248] [Q&A]
I want to output the scan results in csv format  . but how should I only output the few parameters I want

**Comment by bugrakocabay** [comment]
Hi @lijiejie31 , I believe outputting only the flags you have passed is not supported currently. When you use the `-csv` flag, some basic parameters will come along with the flags you have provided.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
